wit/core.wit

```diff
@@ -0,0 +1,74 @@
+package wasmcloud:core@0.1.0
+
+interface types {
+  type link-settings =          list<tuple<string, string>>
+  type host-env-values =        list<tuple<string, string>>
+  type trace-context =          list<tuple<string, string>>
+  type actor-links =            list<link-definition>
+  type cluster-issuers =        list<cluster-issuer-key>
+  type cluster-issuer-key =     string 
+  type capability-contract-id = string
+  type blob =                   list<u8>
+  type public-key =             string
+
+  enum log-level {
```

Comment:
This is already covered in https://github.com/WebAssembly/wasi-logging

Reply:
yea, I was planning on changing this in a subsequent PR since wasi:logging has a critical level and I wan't do discuss if that mattered before I added it